step-release-vis/src/app/components/environment/environment.html

```diff
@@ -27,37 +31,39 @@
         [attr.x]="getLineX() - 1"
         [attr.y]="0"
         [attr.width]="2"
-        [attr.height]="svgHeight - TIMELINE_HEIGHT + 10"
+        [attr.height]="expanded ? svgHeight - TIMELINE_HEIGHT + 10 : svgHeight"
         fill="white"
       />
-      <rect [attr.x]="0" [attr.y]="svgHeight - TIMELINE_HEIGHT + 10" [attr.width]="svgWidth" [attr.height]="2"
-            fill="gray"/>
-      <text
-        *ngFor="let timelinePoint of timelinePoints; let i = index"
-        [attr.x]="timelinePoint.x"
-        [attr.y]="svgHeight - 12"
-        [attr.text-anchor]="getTimelinePointTextAlignment(i)"
-        font-size="12"
-      >
-        {{ timelinePoint.timeString }}
-      </text>
-      <text
-        *ngFor="let timelinePoint of timelinePoints; let i = index"
-        [attr.x]="timelinePoint.x"
-        [attr.y]="svgHeight"
-        [attr.text-anchor]="getTimelinePointTextAlignment(i)"
-        font-size="12"
-      >
-        {{ timelinePoint.dateString }}
-      </text>
-      <line
-        *ngFor="let timelinePoint of timelinePoints"
-        [attr.x1]="timelinePoint.x"
-        [attr.y1]="svgHeight - TIMELINE_HEIGHT + 2"
-        [attr.x2]="timelinePoint.x"
-        [attr.y2]="svgHeight - TIMELINE_HEIGHT / 2 - 3"
-        stroke="gray"
-      />
+      <g id="timeline" *ngIf="expanded">
+        <rect [attr.x]="0" [attr.y]="svgHeight - TIMELINE_HEIGHT + 10" [attr.width]="svgWidth" [attr.height]="2"
+              fill="gray"/>
+        <text
+          *ngFor="let timelinePoint of timelinePoints; let i = index"
+          [attr.x]="timelinePoint.x"
+          [attr.y]="svgHeight - 12"
+          [attr.text-anchor]="getTimelinePointTextAlignment(i)"
+          font-size="12"
+        >
+          {{ timelinePoint.timeString }}
+        </text>
+        <text
+          *ngFor="let timelinePoint of timelinePoints; let i = index"
+          [attr.x]="timelinePoint.x"
+          [attr.y]="svgHeight"
+          [attr.text-anchor]="getTimelinePointTextAlignment(i)"
+          font-size="12"
+        >
+          {{ timelinePoint.dateString }}
+        </text>
+        <line
+          *ngFor="let timelinePoint of timelinePoints"
+          [attr.x1]="timelinePoint.x"
+          [attr.y1]="svgHeight - TIMELINE_HEIGHT + 2"
+          [attr.x2]="timelinePoint.x"
+          [attr.y2]="svgHeight - TIMELINE_HEIGHT / 2 - 3"
```

Comment:
i know it is not new, but can you pls explain the reason for these numbers in a comment?